package.json

```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "lerna run test",
-    "test-rpc": "run-with-testrpc -m \"candy maple cake sugar pudding cream honey rich smooth crumble sweet treat\" --port 8544 --accounts 20 --networkId=9 --gasLimit=10000000 \"lerna run test\" ",
-    "setup": "lerna bootstrap --hoist --no-ci",
+    "test-rpc": "run-with-testrpc -m \"candy maple cake sugar pudding cream honey rich smooth crumble sweet treat\" --port 8544 --accounts 20 --networkId=9 --gasLimit=10000000 -e 100000 \"lerna run test\" ",
+    "setup": "lerna bootstrap --hoist --ci",
```

Comment:
Whats the reason behind that change?

Reply:
In --no-ci mode  `npm run setup` updates packages dependencies. In result PR includes a lot of unrelated package-lock.json. In addition some of this updates breaks build as we has with ethers and recently with ipfs packages

Reply:
Okay, thanks